src/components/CCard.vue

```diff
@@ -182,8 +185,6 @@ const titleClass = computed<string>(() => {
 
   if (props.wrapTitle) classes += " text-wrap";
 
-  if (scrollable.value) classes += " mb-5 mb-md-7";
-
```

Comment:
Why has this been removed? Is this in the correct commit?

Reply:
yes it is, i removed the variable margin because the subtitle now is on top of the divider (shown when scrollable is true) and...